src/server/bundle.ts

```diff
@@ -11,7 +11,7 @@ export interface JSXConfig {
 let esbuildInitialized: boolean | Promise<void> = false;
 async function ensureEsbuildInitialized() {
   if (esbuildInitialized === false) {
-    if (Deno.run === undefined) {
+    if (Deno.Command === undefined) {
```

Comment:
may you consider to check it by `typeof`??

Reply:
I think that in Deno, and other recent js runtime, you cannot anymore re-defined `undefined` and so using typeof is not required. tested in repl
```
$ deno
Deno 1.30.3+e0074b5
exit using ctrl+d, ctrl+c, or close()
REPL is running with all permissions allowed.
To specify permissions, run `deno repl` with allow flags.
> globalThis.undefined = 1
Uncaught TypeError: Cannot assign to read only property 'undefined' of object '#<Window>'
    at <anonymous>:2:22
```
Is there any other reasons to use `typeof` ?